packages/photon/LICENSE

```diff
@@ -0,0 +1,24 @@
+(The MIT License)
```

Comment:
MIT? Does this need to be GPL or GPLv2?

Remove the license and use the package.json property